package.json

```diff
@@ -1,5 +1,10 @@
 {
-    "dependencies": {
-        "yuglify": "^0.1.4"
-    }
+  "dependencies": {
+    "yuglify": "^0.1.4"
+  },
+  "devDependencies": {
+    "gulp": "^3.9.1",
+    "gulp-jshint": "^2.0.0",
+    "jshint": "^2.9.1"
```

Comment:
I was wondering about that, too. Turns out it needs to be installed separately in new versions of gulp-jshint, though I'm not sure why. From the [project page](https://www.npmjs.com/package/gulp-jshint):

> NOTE: as of 2.0 jshint must to be installed with gulp-jshint.